support/do.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+useradd --create-home --shell /bin/bash --user-group deployer
+cp -r /root/.ssh /home/deployer
+chown -R deployer:deployer /home/deployer/.ssh 
+echo 'deployer ALL=(ALL:ALL) ALL' >> /etc/sudoers
```

Comment:
Why not root? I'm sure we could use a shell provisioner to enable root on ubuntu. Every other distro should have a root user right?